engine/frontend/src/component/PackageCatalogForm.js

```diff
@@ -527,7 +535,11 @@ const PackageCatalogForm = ({createEnclave}) => {
             Object.keys(formData).map(key => {
                 const arg = kurtosisPackage.args[key]
                 const argName = getArgName(arg)
-                const type = getType(arg)
+                let type = ""
+                try {
```

Comment:
jus curious what's the reasoning behind wrapping this in a `try catch` if you don't handle the error/do anything on the catch?

Reply:
The function will throw an exception and I don't really care what the error is as it will be due to data missing. So in the absence of the data I fill it in with a default that wont break the UI.